vsce/client/src/util/spawnUpdateIde.ts

```diff
@@ -0,0 +1,29 @@
+import { spawn } from 'child_process';
+
+export default (
+	pathToScript: string,
+	versionCompareJson: VersionCompareJson
+): Promise<number> => {
+	const { dockerC, dockerH, script } = versionCompareJson;
+	console.info('dockerC', dockerC);
+	console.info('dockerH', dockerH);
+	console.info('script', script);
+
+	const argsArray: string[] = [
+		'update-ide',
+		`--json=${dockerC}`,
+		'--rm-json'
+	];
+
+	// --script should only be used when "script": true
```

Comment:
If `"noDiff": true`, should we display a message saying the `reach` script might still be out of date?

Reply:
I think you should replicate the -- `reach update` behavior I linked above. Lines 2192-2194.

In other words: I'd assume you have to replace the script because we can't check for a mismatch and you've already shown them a "this might replace your script" message in the video.

Reply:
So, we want something like this?

```ts
if (noDiff) {
	execSync(
		`curl https://docs.reach.sh/reach -o "${
			pathToScript
		}" && chmod +x "${pathToScript}"`
	);
}
```

Reply:
[f657767](https://github.com/reach-sh/reach-lang/pull/655/commits/f65776716744ef874de5b4f67a56e83e6e69cf0a) should address this.